sdcm/cluster_k8s/minikube.py

```diff
@@ -141,35 +301,159 @@ def local_kubectl_version(self):  # pylint: disable=no-self-use
         #   Client Version: v1.18.5
         return LOCALRUNNER.run("kubectl version --client --short").stdout.rsplit(None, 1)[-1][1:]
 
+    def docker_pull(self, image):
+        LOGGER.info("Pull `%s' to Minikube' Docker environment", image)
+        self.remoter.run(f"docker pull -q {image}")
+
+    @property
+    def remoter(self) -> CommandRunner:
+        return self.host_node.remoter
+
+    def deploy_scylla_manager(self, pool_name: str = None) -> None:
+        self.deploy_minio_s3_backend()
+        super().deploy_scylla_manager(pool_name=pool_name)
+
+    @property
+    @abc.abstractmethod
+    def _create_kubectl_config_cmd(self):
+        pass
+
+    @property
+    @abc.abstractmethod
+    def _target_user(self) -> Optional[str]:
+        pass
+
+    @property
+    @abc.abstractmethod
+    def k8s_server_url(self):
+        pass
+
+    @property
+    @abc.abstractmethod
+    def host_node(self) -> 'BaseNode':
+        """
+        Host where minikube is running
+        """
+        pass
+
+    @property
+    @abc.abstractmethod
+    def is_k8s_software_installed(self) -> bool:
+        """
+        Is kind/k3d/minikube installed ?
+        """
+        pass
+
+    @property
+    @abc.abstractmethod
+    def is_k8s_software_running(self) -> bool:
+        """
+        Is kind/k3d/minikube running ?
+        """
+        pass
+
+    @property
+    @abc.abstractmethod
+    def setup_k8s_software(self):
+        """
+        install kind/k3d/minikube
+        """
+        pass
+
+    @property
+    @abc.abstractmethod
+    def start_k8s_software(self):
+        """
+        Start kind/k3d/minikube
+        """
+        pass
+
+    def deploy(self):
+        if cluster.Setup.REUSE_CLUSTER:
+            if self.is_docker_installed and \
+                    self.is_kubectl_installed and \
+                    self.is_k8s_software_installed and \
+                    self.is_k8s_software_running:
+                self.create_kubectl_config()
+                return
```

Comment:
Please, see this: https://www.python.org/dev/peps/pep-0008/

It is better to not use backslashes if possible and use "and/or" operator on new line to show that it is condition continuation:

```
if cluster.Setup.REUSE_CLUSTER:
    if (self.is_docker_installed
            and self.is_kubectl_installed
            and self.is_k8s_software_installed
            and self.is_k8s_software_running):
        self.create_kubectl_config()
        continue
    raise RuntimeError("SCT_REUSE_CLUSTER is set, but target host is not ready")
```

Reply:
fixed